app/forms/idv/api_document_verification_status_form.rb

```diff
@@ -0,0 +1,45 @@
+module Idv
+  class ApiDocumentVerificationStatusForm
+    include ActiveModel::Model
+    include ActionView::Helpers::TranslationHelper
+
+    validate :throttle_if_rate_limited
+    validate :timeout_error
+    validate :failed_result
+
+    def initialize(async_state:, document_capture_session:)
+      @async_state = async_state
+      @document_capture_session = document_capture_session
+    end
+
+    def submit
+      FormResponse.new(
+        success: valid?,
+        errors: errors.messages,
+        extra: {
+          remaining_attempts: remaining_attempts,
+        },
+      )
+    end
+
+    def remaining_attempts
+      return unless @document_capture_session
+      Throttler::RemainingCount.call(@document_capture_session.user_id, :idv_acuant)
+    end
+
+    def throttle_if_rate_limited
```

Comment:
Why are we throttling status?  It just does a db read.

Reply:
>Why are we throttling status? It just does a db read.

It doesn't actually increment the throttler here, only checking if already throttled. But you're right, it probably doesn't make much sense to need it at all.

Reply:
> Why are we throttling status? It just does a db read.

Removed throttle validation in 2ff4bb682.